spec/models/restaurant_spec.rb

```diff
@@ -3,10 +3,18 @@
 RSpec.describe Restaurant, type: :model do
   describe 'DB Table' do
     it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :city }
+    it { is_expected.to have_db_column :post_code }
+    it { is_expected.to have_db_column :street_address }
+    it { is_expected.to have_db_column :longitude }
+    it { is_expected.to have_db_column :latitude }
   end
 
   describe 'Validations' do
     it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_presence_of :city }
+    it { is_expected.to validate_presence_of :post_code }
+    it { is_expected.to validate_presence_of :street_address }
```

Comment:
I would like you to include a spec for `#full_address` method. At the very least a `responds_to` spec

Reply:
Spec included, not sure if included in a good way though

Reply:
I see no spec for full_address.

Reply:
At the bottom of the file

Reply:
Alfred, that is `street_address` right? I'm asking for unit tests for the `full_address` method you have on your model. Please read my comments carefully.